deploy.go

```diff
@@ -676,6 +738,64 @@ func (gc *GatewayClient) CheckForCompletionQueueCommands(currentPhase string) (*
 	return &gatewayResponse, nil
 }
 
+// UninstallCluster used for start installation
```

Comment:
I did not understand this comment. UninstallCluster is for starting installation?

Reply:
Updated.